### ISSUE TITLE
There's no obvious way for a screen reader user to get the version of an installed extension

### ISSUE THREAD

**Issue by camlorn** [microsoft/vscode#102224]
- VSCode Version: 1.47.0
- OS Version: Windows 10 X64

Steps to Reproduce:

1. Open show extensions
2. Find any extension in the list

Screen readers say "C/C++. Press enter for extension details. This extension is enabled globally".  To get the version, publisher, etc. you have to leave focus mode (NVDA term) or copy the extension.  Enter opens the README.  This label should probably be "C/C++, 1.2.3, Microsoft, press enter for extension details" I'm leaving out the field names because the fields are entirely self-explanatory, but maybe there should be some.

Does this issue occur when all extensions are disabled?: No, but that's because it removes extensions from the list. But note that the recommended extension list has the same problem.

**Comment by isidorn** [comment]
@camlorn thanks a lot for your feedback. I have updated the aria label of the extensions based on your proposal.
You can try it out with vscode insiders from Wednesday and let us know how ti works for you. Thanks!
